template/python3-flask-debian/index.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Alex Ellis 2017. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+from flask import Flask, request
+from function import handler
+#from gevent.wsgi import WSGIServer
+from gevent.pywsgi import WSGIServer
+
+app = Flask(__name__)
+
+@app.before_request
+def fix_transfer_encoding():
+    """
+    Sets the "wsgi.input_terminated" environment flag, thus enabling
+    Werkzeug to pass chunked requests as streams.  The gunicorn server
+    should set this, but it's not yet been implemented.
+    """
+
+    transfer_encoding = request.headers.get("Transfer-Encoding", None)
+    if transfer_encoding == u"chunked":
+        request.environ["wsgi.input_terminated"] = True
+
+@app.route("/", defaults={"path": ""}, methods=["POST", "GET"])
+@app.route("/<path:path>", methods=["POST", "GET"])
+def main_route(path):
+    ret = handler.handle(request.get_data())
+    return ret
+
+if __name__ == '__main__':
+
```

Comment:
Since we are using gevent base wsgi server, we should monkey patch standard libraries to make it cooperative with gevent.

```
from gevent import monkey
monkey.patch_all()
```
http://www.gevent.org/api/gevent.monkey.html

Reply:
... well, ok.  

But that kind of represents a change to all of the templates, which is out of the scope of this PR>

That's probably better as a new issue, especially because I at least don't know any implications for using a newer version of gevent..

Reply:
Can you explain what "we should money patch" means? 🤔  I would say this is also out of scope since the task was to port from Alpine to Debian. Happy for you to raise an issue Vivek if you can explain in a bit more detail there?

Reply:
Gevent is built on top of libevent/libev (for asynchronous I/O) and greenlets (lightweight cooperative multi-threading). Greenlets are based on Stackless Python. Greenlets has no implicit scheduling and we can control exactly when our code runs.

In case of I/O block, the code will automatically switch to other I/O or wait for acknowledgment, however the standard libraries implementation does not work that way. 

Monkey patching is required to patch the standard library with compatible cooperative counterparts from gevent package.

Reply:
I agree that, this is out of scope of this PR. I will raise a separate issue.